### PR TITLE
Add enum for deployment phase strategy in config.json.

### PR DESCRIPTION
**Pull request by ssk2**
@triclambert, you mentioned that this should be an enum last week. Here's the fix!

### CODE REVIEW

**Comment by gabrielhartmann** [comment]
@ssk2:  Why only Kafka?

**Comment by ssk2** [comment]
@gabrielhartmann are there others? I thought Kafka was the only one that has this option.

**Comment by ssk2** [comment]
@gabrielhartmann, updated to cover the other cases I've found.

**Comment by ssk2** [comment]
retest this please

File: frameworks/elastic/universe/config.json [hunk 1]
```diff
@@ -28,7 +28,8 @@
         "deploy_strategy": {
           "description": "Elasticsearch deploy strategy. [parallel, serial]",
           "type": "string",
-          "default": "parallel"
+          "default": "parallel",
+          "enum" : ["parallel"]
```
Comment (loren) [inline frameworks/elastic/universe/config.json]: `serial`, too.

Comment (ssk2) [inline frameworks/elastic/universe/config.json]: Whoops, thanks.

**Pull request merged**